labelbox/data/serialization/coco/instance_dataset.py

```diff
@@ -27,6 +28,14 @@ def mask_to_coco_object_annotation(annotation: ObjectAnnotation, annot_idx: int,
     if shapely.type == 'Polygon':
         shapely = [shapely]
 
+    classifications = {}
+    for classification in annotation.classifications:
+        if classification.name == "is_pickable":
+            is_pickable = bool(strtobool(classification.value.answer.name))
+            classifications[classification.name] = is_pickable
+        else:
+            classifications[classification.name]: classification.value.answer.name
+
```

Comment:
A suggestion to make the serialisation of our custom attributes/classifications more declarative (and hopefully more obvious), based on the tenuous assumption that we may need to add more in the future.

I'm not a fan of proliferating if...elif blocks in situations like this, when the number of special cases is unbounded. Having said that, right now we have only 1 case so this is probably a case of premature refactoring. So take it or leave it!

```suggestion
    # Serialisers for named custom attributes
    value_serialisers = defaultdict(
        lambda: lambda s: s,
        is_pickable=lambda s: bool(strtobool(s)),
    )

    classifications = {
        c.name: value_serialisers[c.value.answer.name] for c in annotation.classifications
    }
```